doc/usermanual/preferences/keyaccelerators.xml

```diff
@@ -24,6 +24,11 @@
     In order to remove an existing keyboard shortcut, click on the action and press backspace.
   </para>
 
+  <para>
+    In addition to classic shortcuts, you can set dynamic shortcuts to some darkroom module sliders.
+    This allow you to change the value of the mapped slider with <emphasis>key + mouse scroll</emphasis>.
```

Comment:
This allows you to change the value of the mapped slider with <emphasis>key + mouse scroll</emphasis>.